src/azure-cli/azure/cli/command_modules/containerapp/_params.py

```diff
@@ -0,0 +1,480 @@
+
+    with self.argument_context('containerapp ssl upload') as c:
+        c.argument('hostname', help='The custom domain name.')
+        c.argument('environment', options_list=['--environment', '-e'], help='Name or resource id of the Container App environment.')
+        c.argument('certificate_file', options_list=['--certificate-file', '-f'], help='The filepath of the .pfx or .pem file')
+        c.argument('certificate_password', options_list=['--password', '-p'], help='The certificate file password')
+        c.argument('certificate_name', options_list=['--certificate-name', '-c'], help='Name of the certificate which should be unique within the Container Apps environment.')
+
+    with self.argument_context('containerapp hostname bind') as c:
+        c.argument('hostname', help='The custom domain name.')
+        c.argument('thumbprint', options_list=['--thumbprint', '-t'], help='Thumbprint of the certificate.')
+        c.argument('certificate', options_list=['--certificate', '-c'], help='Name or resource id of the certificate.')
+        c.argument('environment', options_list=['--environment', '-e'], help='Name or resource id of the Container App environment.')
+        c.argument('validation_method', options_list=['--validation-method', '-v'], help='Validation method of custom domain ownership.')
+
+    with self.argument_context('containerapp hostname add') as c:
+        c.argument('hostname', help='The custom domain name.')
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+
+    with self.argument_context('containerapp hostname list') as c:
+        c.argument('name', id_part=None)
+        c.argument('location', arg_type=get_location_type(self.cli_ctx))
+
+    with self.argument_context('containerapp hostname delete') as c:
+        c.argument('hostname', help='The custom domain name.')
+
+    # Compose
+    with self.argument_context('containerapp compose create') as c:
+        c.argument('environment', options_list=['--environment', '-e'], help='Name or resource id of the Container App environment.')
+        c.argument('compose_file_path', options_list=['--compose-file-path', '-f'], help='Path to a Docker Compose file with the configuration to import to Azure Container Apps.')
+        c.argument('transport_mapping', options_list=['--transport-mapping', c.deprecate(target='--transport', redirect='--transport-mapping')], action='append', nargs='+', help="Transport options per Container App instance (servicename=transportsetting).")
+
+    with self.argument_context('containerapp env workload-profile') as c:
+        c.argument('env_name', options_list=['--name', '-n'], help="The name of the Container App environment")
+        c.argument('workload_profile_name', options_list=['--workload-profile-name', '-w'], help='The friendly name for the workload profile')
+
+    with self.argument_context('containerapp env workload-profile set') as c:
```

Comment:
do we have `workload-profile set` in previous extension? seems dupe with `add` and `update`?

Reply:
g.custom_command('set', 'set_workload_profile', deprecate_info=g.deprecate(hide=True))

I will remove this argument.